docs/manual-indexes.adoc

```diff
@@ -0,0 +1,166 @@
+= Manual Indexes
+
+== Data Used
+
+In the below examples I used the flight data setup from the following link:
+
+https://github.com/nicolewhite/neo4j-flights
+
+
+== Enable Manual Indexes on Node and Relationship Property
```

Comment:
Actually those are about auto-indexes, which are a kind of legacy-index but not the ones I was focusing on in apoc. Apoc has it's own procedures for adding things to the indexes which are meant to be called in batch post-creation or better as par of the create statements.

I'd rather prefer not to have legacy-autoindexes.